<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>236</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QPushButton" name="playButton">
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border-radius: 50%; /* make button circular */
	background-color: rgb(88, 175, 158);
    color: white; /* text color */
    padding: 10px 20px; /* adjust padding for better appearance */
}

QPushButton:hover {
    background-color: #45a049; /* darker green on hover */
}

QPushButton:pressed {
    background-color: #367c3a; /* even darker green when pressed */
}</string>
      </property>
      <property name="text">
       <string>Play</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="stopButton">
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border-radius: 50%; /* make button circular */
    background-color: rgb(88, 175, 158);
    color: white; /* text color */
    padding: 10px 20px; /* adjust padding for better appearance */
}

QPushButton:hover {
    background-color: #1e88e5; /* darker blue on hover */
}

QPushButton:pressed {
    background-color: #1976d2; /* even darker blue when pressed */
}</string>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pauseButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border-radius: 50%; /* make button circular */
    background-color: rgb(88, 175, 158);
    color: white; /* text color */
    padding: 10px 20px; /* adjust padding for better appearance */
}

QPushButton:hover {
    background-color: #d32f2f; /* darker red on hover */
}

QPushButton:pressed {
    background-color: #b71c1c; /* even darker red when pressed */
}</string>
      </property>
      <property name="text">
       <string>Pause</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
